static/styles/message.css

```diff
@@ -0,0 +1,128 @@
+$first_column_width: 46px;
+$distance_of_text_elements_from_message_box_top: 8px;
+
+.message_row {
```

Comment:
From a readability standpoint, would it make sense to do `&.include_sender` variations after the base CSS that it overrides?

Reply:
Ok, done!